CONTRIBUTING.md

```diff
@@ -1,75 +1,47 @@
-# Welcome to Reflex contributing guide! 🥳
+# Reflex Contributing Guidelines
 
-## Getting started
+For an extensive guide on the different ways to contribute to Reflex please visite our notion's [Contributing Guide](https://www.notion.so/reflex-dev/2107ab2bc166497db951b8d742748284?v=f0eaff78fa984b5ab15d204af58907d7).
```

Comment:
typo: "visit our Notion's". I think we can reword this also to "see our Contributing Guide on Notion"